UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/BodyPartBlood.cs

```diff
@@ -15,91 +15,91 @@ public partial class BodyPart
 		/// Modifier that multiplicatively reduces the efficiency of the body part based on damage
```

Comment:
Unrelated to your changes, but all of these partial parts of BodyPart should die and become a System/Component instead